addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -88,11 +89,40 @@ class MessageList extends Component {
                 threadViewer: threadView && threadView.threadViewer,
             };
         });
+        /**
+         * This observer will emit an hit when the composer is resized by any
+         * mean.
+         */
+        this.resizeObserver = new ResizeObserver(elements => {
```

Comment:
Would explain what this is (some "hook" that is triggered whenever the root node is being resized), and ignore the explanation about the cause (composer resize)... Or could be in separated line just to give some context/example this message list is resized after a full OWL (re-)render, but considering it's technical for bad reasons, better explain it as some extra info.